Install/install.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash -e
+
+GIT_REPO=https://github.com/TomWhitwell/SlowMovie
+GIT_BRANCH=master
+PYTHON_VERSION=3
+SKIP_DEPS=false
+
+# color code variables
+RED="\e[0;91m"
+YELLOW="\e[0;93m"
+RESET="\e[0m"
+
+function install_linux_packages(){
+  sudo apt-get update
+
+  sudo apt-get install -y ffmpeg git python3-pip
+}
+
+function install_python_packages(){
+
+
+  sudo pip3 install setuptools -U
+  sudo pip3 install -r $LOCAL_DIR/Install/requirements.txt
+}
+
+function build_python_libraries(){
+  WAVESHARE_DIR=/home/pi/e-Paper
+
+  if [ -d "${WAVESHARE_DIR}" ]; then
+    echo -e "Updating Waveshare Drivers"
+    cd $WAVESHARE_DIR
+    git fetch
+    git pull
+  else
+    echo -e "Installing Waveshare Drivers"
+    git clone https://github.com/waveshare/e-Paper
+  fi
+
+  echo -e "${YELLOW}Be patient — this takes time${RESET}"
+  cd $WAVESHARE_DIR/RaspberryPi_JetsonNano/python/
+  sudo python3 setup.py install
+
+  # return to home directory
+  cd /home/pi
+}
+
+function setup_hardware(){
+  echo "Setting up SPI"
+  if ls /dev/spi* &> /dev/null; then
+      echo -e "SPI already enabled"
+  else
+      if command -v raspi-config > /dev/null && sudo raspi-config nonint get_spi | grep -q "1"; then
+          sudo raspi-config nonint do_spi 0
+          echo -e "SPI is now enabled"
+      else
+          echo -e "${RED}There was an error enabling SPI, enable manually with sudo raspi-config${RESET}"
+      fi
+  fi
+}
+
+function install_slowmovie(){
+  FIRST_TIME=1  # if this is a first time install
+
+  if [ "$SKIP_DEPS" = false ]; then
+    # install from apt
+    install_linux_packages
+
+    # configure the hardware
+    setup_hardware
+  else
+    echo -e "Skipping dependency installs, updating SlowMovie code only"
+  fi
+
+  if [ -d "${LOCAL_DIR}" ]; then
+    echo -e "Existing Install Found - Updating Repo"
+    cd $LOCAL_DIR
+    git fetch
+    git checkout $GIT_BRANCH
+    git pull
+
```

Comment:
```suggestion

# Update service if installed and file changed
if [ -f "/etc/systemd/system/slowmovie.service" ] && ! (cmp -s "slowmovie.service" "/etc/systemd/system/slowmovie.service"); then
  sudo cp slowmovie.service /etc/systemd/system
  sudo systemctl daemon-reload
fi

```
I don't see slowmovie.service changing any more, but just in case.

Reply:
I did add this but made [some modifications](https://github.com/robweber/SlowMovie/blob/install_script/Install/install.sh#L112).